File: 2.0/java/webhooks/getWebhook.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.webhooks.model.Webhook;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api login", "api password");
        Webhook webhook = client.webhooksApi().get(379506003L, "name,callback");
    }
}

File: 2.0/csharp/webhooks/getWebhook.cs
using CallfireApiClient;
using CallfireApiClient.Api.Webhooks.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        Webhook webhook = client.WebhooksApi.Get(379506003, "name,callback");
    }
}
